scene/2d/tile_map.cpp

```diff
@@ -3555,6 +3555,159 @@ Rect2 TileMap::_edit_get_rect() const {
 	const_cast<TileMap *>(this)->item_rect_changed(any_changed);
 	return rect;
 }
+
```

Comment:
I don't think this code belongs here. This should be part of the editor, not from the TileMap node.